tests/e2e/browser/音声詳細.spec.ts

```diff
@@ -8,6 +8,45 @@ test.beforeEach(async ({ page }) => {
   await page.goto(BASE_URL);
 });
 
+function getNthAccentPhraseInput({ page, n }: { page: Page; n: number }) {
+  return page.getByLabel(`${n + 1}番目のアクセント区間の読み`);
+}
+
+test("単体アクセント句の読み変更", async ({ page }) => {
+  await navigateToMain(page);
+  await page.waitForTimeout(100);
+
+  const textField = page.getByRole("textbox", { name: "1行目" });
+  await textField.click();
+  await textField.fill("1234");
+  await textField.press("Enter");
+
+  const inputs = Array.from({ length: 4 }, (_, i) =>
+    getNthAccentPhraseInput({ page, n: i })
+  );
+
+  await page.getByText("ヨ", { exact: true }).click();
+  await inputs[3].fill("ヨン,、,、");
+  await inputs[3].press("Enter");
+
+  await page.getByText("ジュ", { exact: true }).click();
+  await inputs[2].fill("サンジュウ,、,、");
+  await inputs[2].press("Enter");
+
+  await page.getByText("ヒャ", { exact: true }).click();
+  await inputs[1].fill("ニヒャク、");
+  await inputs[1].press("Enter");
+
+  await page.getByText("セ", { exact: true }).click();
+  await inputs[0].fill("セン,");
+  await inputs[0].press("Enter");
```

Comment:
僕は同一PRなのでわかりますが、それぞれ意図をメモっておくと意図が伝わりやすいかもと思いました！
こんな感じとかでしょうか。

```suggestion
  // 最後のアクセント区間に読点をつけても無視される
  await page.getByText("ヨ", { exact: true }).click();
  await inputs[3].fill("ヨン,、,、");
  await inputs[3].press("Enter");

  // 連続する読点を追加すると１つに集約される
  await page.getByText("ジュ", { exact: true }).click();
  await inputs[2].fill("サンジュウ,、,、");
  await inputs[2].press("Enter");

  // 読点を追加
  await page.getByText("ヒャ", { exact: true }).click();
  await inputs[1].fill("ニヒャク、");
  await inputs[1].press("Enter");

  await page.getByText("セ", { exact: true }).click();
  await inputs[0].fill("セン,");
  await inputs[0].press("Enter");
```

Reply:
> expectがちょっと遠いので各々の操作の直後にくっつけてあげるとよりわかりやすいかもです。

「ニヒャク、」直後にexpectすると多分入力欄が残ってて反応するのでwaitForTimeout()必要そうなんですよね。
適当な値が分からなかったので確実な実装の方にしてしまいました。
500ms待てば十分でしょうか…？

Reply:
なるほどです。100msくらいで大丈夫かなと！